source/msam/test/test_settings.py

```diff
@@ -20,29 +20,27 @@ class TestSettings(unittest.TestCase):
     This class extends TestCase with testing functions
     """
 
+    def setUp(self):
+        from chalicelib import settings
+        settings.DYNAMO_RESOURCE.reset_mock()
+        settings.DYNAMO_RESOURCE.Table.return_value.put_item.reset_mock()
+
     def test_put_setting(self, patched_env, patched_resource,
                              patched_client):
         """
         Test the put_setting function
         """
         from chalicelib import settings
         settings.put_setting(KEY, VALUE)
+        settings.DYNAMO_RESOURCE.Table.return_value.put_item.assert_called_once_with(Item={"id": KEY, "value": VALUE})
 
     def test_get_setting(self, patched_env, patched_resource, patched_client):
         """
         Test the get_setting function
         """
         from chalicelib import settings
         settings.get_setting(KEY)
-
-        mock_table = MagicMock()
-        mock_table.get_item.return_value = {"Item": {"value": VALUE}}
-        patched_resource.return_value.Table.return_value = mock_table
-        settings.get_setting(KEY)
-
-        mock_table.get_item.side_effect = CLIENT_ERROR
```

Comment:
does this scenario (CLIENT_ERROR) also need to be rewritten?

Reply:
good catch, I have written tests to cover those deleted cases

Reply:
@sandimciin check the new lines after the changes here